toolkit/resources/manifests/package/toolchain_x86_64.txt

```diff
@@ -9,9 +9,9 @@ bash-5.1.8-3.cm2.x86_64.rpm
 bash-debuginfo-5.1.8-3.cm2.x86_64.rpm
 bash-devel-5.1.8-3.cm2.x86_64.rpm
 bash-lang-5.1.8-3.cm2.x86_64.rpm
-binutils-2.37-5.cm2.x86_64.rpm
-binutils-debuginfo-2.37-5.cm2.x86_64.rpm
-binutils-devel-2.37-5.cm2.x86_64.rpm
+binutils-2.40-1.cm2.x86_64.rpm
+binutils-debuginfo-2.40-1.cm2.x86_64.rpm
+binutils-devel-2.40-1.cm2.x86_64.rpm
```

Comment:
please add the binutils-gprofng subpackage to these toolchain_* manifests

Reply:
Added